src/webhook/validation/api_url.go

```diff
@@ -9,12 +11,26 @@ const (
 	errorNoApiUrl = `The DynaKube's specification is missing the API URL or still has the example value set.
 	Make sure you correctly specify the URL in your custom resource.
 	`
+	noError = ""
 )
 
 func noApiUrl(dv *dynakubeValidator, dynakube *dynatracev1beta1.DynaKube) string {
-	if dynakube.Spec.APIURL != "" && dynakube.Spec.APIURL != exampleApiUrl {
-		return ""
+	apiUrl := dynakube.Spec.APIURL
+
+	if !strings.HasSuffix(apiUrl, "/api") {
+		log.Info("api url does not end with /api", "apiUrl", apiUrl)
+		return errorNoApiUrl
```

Comment:
this should be a separate error checked separately in a function like `incorrectApiUrl` saying: `The Dynakube's API URL does not end on '/api/`

Reply:
the new function has to then be registered in `validation/config.go` like the other validators

Reply:
the check in `TenantUUID()` is redundant then, as it's impossible to create a Dynakube that does not fulfill these conditions.

Reply:
OK, moved the API URL validation to webhook code (look for `isInvalidApiUrl()`)